docs/release_notes.rst

```diff
@@ -0,0 +1,3 @@
+=====================================================
```

Comment:
What sort of thing is going to go here?

Reply:
I'm thinking of some sort of changelog, writing the changes included in a particular version... since we'll be using git tags/releases, we could redirect to that page (https://github.com/pi-top/pi-top-Python-SDK/releases) ... or just remove this section

Reply:
I was thinking that this could be the sort of place where we explain how to upgrade from earlier versions of the libraries, and where we expect possible breakage in the future? Not sure that completely maps to the title, but I think we should start with that and see how it develops.